pkg/download/classic/downloader_test.go

```diff
@@ -14,14 +14,16 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package classic
+package classic_test
 
 import (
 	"context"
 	"fmt"
+	"github.com/dynatrace/dynatrace-configuration-as-code/v2/cmd/monaco/download"
```

Comment:
even for tests, I don't think a package should ever import from cmd, please remove that (think it's to access the feature flag convenience method, IMO that one's not needed for testing. Currently we assume they are ON so keeping the tests at default filtering or where needed just explicitly setting WithFiltering(true) or WithFiltering(false) should be enough. 

Testing that the flags control the option should be done on the cmd package integration test IMO.